WindowsServerDocs/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs.md

```diff
@@ -218,6 +221,11 @@ This will ensure that all data trasmitted to fs.contoso.com end up at the ILB an
 
 In order to ensure that Web Application Proxy servers are able to reach the AD FS servers behind the ILB, create a record in the %systemroot%\system32\drivers\etc\hosts for the ILB. Note that the distinguished name (DN) should be the federation service name, for example fs.contoso.com. And the IP entry should be that of the ILB's IP address (10.3.0.8 as in the example).
 
+> [!WARNING]
+>If you are using the WID (Windows Internal Database) for your AD FS database, this value should instead be temporarily set to point to your primary AD FS server or the Web Application Proxy will fail enrollement. After you have successfully enrolled all Web Appplication Proxy servers, change this DNS entry to point to the load balancer.
+>
+>
```

Comment:
As above, codestyle/consistency.
```suggestion
> If you are using the WID (Windows Internal Database) for your AD FS database, this value should instead be temporarily set to point to your primary AD FS server or the Web Application Proxy will fail enrollement. After you have successfully enrolled all Web Appplication Proxy servers, change this DNS entry to point to the load balancer.

```
(plus removal of redundant blank line and redundant MD indent markers)